plottable.d.ts

```diff
@@ -277,6 +277,19 @@ declare module Plottable {
 
 
 declare module Plottable {
+    module Utils {
+        class CallbackSet<CB extends Function> {
+            constructor();
+            add(value: CB): CallbackSet<CB>;
+            remove(value: CB): CallbackSet<CB>;
```

Comment:
Chaining is something that is pretty consistent in our codebase, and we should press for consistency where we can.  I think one thing that is really consistent with our library and other modern libraries is the idea of returning `this` whenever we are explicitly setting state / otherwise would return a `void`